### PR TITLE
move importing to celery

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: molo/core/content_import/views.py [hunk 1]
```diff
@@ -64,7 +65,11 @@ def import_content(request):
             'warnings': result['warnings']
         })
     else:
-        api.import_content(repos, locales)
+        tasks.import_content.delay(
+            data,
+            request.user.username,
+            request.user.email,
+            request.get_host())
         return Response(status=200, data={
```
Comment (justinvdm) [inline molo/core/content_import/views.py]: Bikeshed: can we use `201` here?

File: molo/core/tasks.py [hunk 2]
```diff
@@ -28,3 +32,27 @@ def rotate_content():
...
+
+Regards,
+Molo Team
+"""
+
+    send_mail(
+        settings.CONTENT_IMPORT_SUBJECT,
+        message % {'name': username, 'host': host},
+        settings.FROM_EMAIL,
+        [email], fail_silently=False)
```
Comment (justinvdm) [inline molo/core/tasks.py]: We need a test for it. Maybe in a new ticket if this is urgent?

File: molo/core/content_import/views.py (hunk 1 above)
Comment (miltontony) [inline molo/core/content_import/views.py]: I was going to ask about that - I saw you updated your fixtures with 201

Comment (justinvdm) [inline molo/core/content_import/views.py]: Doh, `202` was the code I was looking for.

**Comment by justinvdm** [comment]
:+1:

File: molo/core/tasks.py (hunk 2 above)
Comment (miltontony) [inline molo/core/tasks.py]: :ok_hand: issue raise #159

**Pull request merged**